motion/hybridpath_controller/hybridpath_controller/adaptive_backstep.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+class AdaptiveBackstep:
+    def __init__(self, kappa = 0.5):
+        self.init_system(kappa)
+
+    def init_system(self, kappa):
+
+        I = np.eye(3)
+
+        K_1 = np.diag([10, 10, 10])
+        self.K_1_tilde = K_1 + kappa*I
+        self.K_2 = np.diag([60, 60, 30])
+        self.tau_max = np.array([41.0, 50.0, 55.0]) # Må tilpasses thrusterne
+
+        ## Forenklet modell ## Bør også endres
+        m = 50
+        self.M = np.diag([m, m, m])
+        self.D = np.diag([10, 10, 5])
+
+    def control_law(self, eta, nu, w, v_ref, dt_v_ref, dtheta_v_ref, eta_d, dtheta_eta_d, ddtheta_eta_d): # dtheta == ds
```

Comment:
Same story as the "too-many-return-values-of-the-same-type", it is very easy for a user to mess up this call. If there is a simple way of packing these into a data struct, I would highly suggest doing so.